src/main/typescript/lib/files/files.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2018 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FileInfo, Files } from "../../model/FileInfo"
+
+type FileWithPath = FileInfo & { dirpath: string }
+
+export const filesConverter: (input: any) => Files = input => {
```

Comment:
Could you please at some tests, equivalent to [`lib/metadata`](https://github.com/DANS-KNAW/easy-deposit-ui/tree/a987c4b62b42d523bd725277c328f085fcf654f3/src/test/typescript/lib/metadata)?